pom.xml

```diff
@@ -44,31 +44,17 @@
                 <directory>src/main/resources</directory>
                 <filtering>true</filtering>
                 <includes>
-                    <include>paper-plugin.yml</include>
```

Comment:
This plugin only works on Paper and derivatives, because it makes use of the `PaperServerListPingEvent`, which does not exist in Spigot. Having only the `paper-plugin.yml` instead of the `plugin.yml` prevents this plugin from even loading on Spigot servers.